src/Sylius/Bundle/ShopBundle/Resources/views/Common/Order/Table/_shipping.html.twig

```diff
@@ -1,25 +1,22 @@
 {% import "@SyliusShop/Common/Macro/money.html.twig" as money %}
 
 {% set orderShippingPromotionAdjustment = constant('Sylius\\Component\\Core\\Model\\AdjustmentInterface::ORDER_SHIPPING_PROMOTION_ADJUSTMENT') %}
+{% set shippingAdjustment = constant('Sylius\\Component\\Core\\Model\\AdjustmentInterface::SHIPPING_ADJUSTMENT') %}
 {% set orderShippingPromotions = sylius_aggregate_adjustments(order.getAdjustmentsRecursively(orderShippingPromotionAdjustment)) %}
 
 {% if not orderShippingPromotions is empty %}
     <td colspan="2" id="promotion-shipping-discounts">
-        <div class="ui relaxed divided list">
-            {% for label, amount in orderShippingPromotions %}
-                <div class="item">
-                    <div class="content">
-                        <div class="header">{{ label }}</div>
-                        <div class="description">
-                            {{ money.convertAndFormat(amount) }}
-                        </div>
-                    </div>
-                </div>
-            {% endfor %}
-        </div>
+        <div><strong>{{ 'sylius.ui.shipping'|trans }}:</strong></div>
+        <span class="old-price">{{ money.convertAndFormat(order.getAdjustmentsTotal(shippingAdjustment)) }}</span>
+        <span>
+            {{ money.format(order.getAdjustmentsTotal(shippingAdjustment) + order.getAdjustmentsTotal(orderShippingPromotionAdjustment), order.currencyCode) }}
+            <i id="shipping-promotion-details" class="question circle icon popup-js"
+               data-html="{% for label, amount in orderShippingPromotions %}<div>{{ label }}: {{ money.format(amount, order.currencyCode) }}</div>{% endfor %}"
+            ></i>
+        </span>
     </td>
 {% endif %}
 <td colspan="{% if not orderShippingPromotions is empty %}2{% else %}4{% endif %}" class="right aligned" id="shipping-total">
     {{ 'sylius.ui.shipping_total'|trans }}:
-    {{ money.format(order.shippingTotal, order.currencyCode) }}
+    {{ money.convertAndFormat(order.shippingTotal) }}
```

Comment:
Why it has been changed? Why do we use both `money.convertAndFormat` and `money.format` on the same page?

Reply:
Added more information on that in review's comment.

Reply:
i don't agree with you, in my opinion customer should see his currency and only on summary page and payment label he should see order currency.
like here
![Screenshot 2019-06-07 at 14 09 04](https://user-images.githubusercontent.com/29897151/59103110-3ad43f00-892e-11e9-8378-12a1123a7607.png)

Reply:
This is what I meant with "Why do we use both `money.convertAndFormat` and `money.format` on the same page?":

<img width="288" alt="Screenshot 2019-06-10 at 20 04 38" src="https://user-images.githubusercontent.com/1897953/59194274-1640c700-8bbb-11e9-8a75-72a8ecdcd92b.png">
<img width="855" alt="Screenshot 2019-06-10 at 20 04 29" src="https://user-images.githubusercontent.com/1897953/59194275-16d95d80-8bbb-11e9-9034-ed24898d931c.png">
<img width="331" alt="Screenshot 2019-06-10 at 20 04 10" src="https://user-images.githubusercontent.com/1897953/59194276-16d95d80-8bbb-11e9-9db3-75bfb4f81261.png">